### PR TITLE
sonar-scanner: Clarify output and docs, bump version

### PR DESCRIPTION
**Pull request by underyx**

### CODE REVIEW

File: sonar-scanner/bin/scan [hunk 1]
```diff
@@ -3,10 +3,14 @@ echo
 echo kiwicom/sonar-scanner • scan mode
 echo =================================
 echo This is a wrapper for \`$ sonar-scanner\` with default arguments to run in GitLab CI
-echo It takes one argument: a comma-separated of directories to check, such as: \`$ scan kw,cron,scripts\`
-echo If you want to customize it, run \`$ sonar-scanner \<your-arguments\>\` instead.
+echo It takes one one argument: a comma-separated of directories to check, such as: \`$ preview kw,test\`
```
Comment (aexvir) [inline sonar-scanner/bin/scan]: one + one = two

File: sonar-scanner/bin/preview [hunk 2]
```diff
@@ -3,10 +3,14 @@ echo
 echo kiwicom/sonar-scanner • preview mode
 echo ====================================
 echo This is a wrapper for \`$ sonar-scanner\` with default arguments to run in GitLab CI
-echo It takes one argument: a comma-separated of directories to check, such as: \`$ preview kw,cron,scripts\`
-echo If you want to customize it, run \`$ sonar-scanner \<your-arguments\>\` instead.
+echo It takes one one argument: a comma-separated of directories to check, such as: \`$ preview kw,test\`
```
Comment (aexvir) [inline sonar-scanner/bin/preview]: one + one = two

Comment (aexvir) [reply sonar-scanner/bin/preview]: A comma-separated (list)? Something is missing here

File: sonar-scanner/bin/scan (hunk 1 above)
Comment (aexvir) [reply sonar-scanner/bin/scan]: Same as before

File: sonar-scanner/bin/preview [hunk 3]
```diff
@@ -3,10 +3,14 @@ echo
...
 echo This is a wrapper for \`$ sonar-scanner\` with default arguments to run in GitLab CI
-echo It takes one argument: a comma-separated of directories to check, such as: \`$ preview kw,cron,scripts\`
-echo If you want to customize it, run \`$ sonar-scanner \<your-arguments\>\` instead.
+echo It takes one argument: a comma-separated list of directories to check, such as: \`$ preview kw,test\`
+echo If you want to customize it, add your extra arguments at the end.
+echo The \`sonar-scanner\` binary is also available for fully customized usage.
 echo
 
+echo "sonar-scanner's output"
+echo "======================="
```
Comment (paveldedik) [inline sonar-scanner/bin/preview]: I'd remove one `=` (so that it's aligned with the title)

File: sonar-scanner/bin/scan [hunk 4]
```diff
@@ -3,10 +3,14 @@ echo
...
 echo This is a wrapper for \`$ sonar-scanner\` with default arguments to run in GitLab CI
-echo It takes one argument: a comma-separated of directories to check, such as: \`$ scan kw,cron,scripts\`
-echo If you want to customize it, run \`$ sonar-scanner \<your-arguments\>\` instead.
+echo It takes one argument: a comma-separated list of directories to check, such as: \`$ preview kw,test\`
+echo If you want to customize it, add your extra arguments at the end.
+echo The \`sonar-scanner\` binary is also available for fully customized usage.
 echo
 
+echo "sonar-scanner's output"
+echo "======================="
```
Comment (paveldedik) [inline sonar-scanner/bin/scan]: Same here.

File: sonar-scanner/bin/scan [hunk 5]
```diff
@@ -21,10 +25,15 @@ sonar-scanner \
...
 if grep -q 'A regular analysis is required before a branch analysis' /tmp/sonar-scanner-errors; then
+    echo
+    echo kiwicom/sonar-scanner
+    echo =====================
     echo
     echo "Oh, so this is the first time I'm running here!"
     echo "Let me try again with a different kind of scan to create your project."
     echo
+    echo "sonar-scanner's output"
+    echo "======================="
```
Comment (paveldedik) [inline sonar-scanner/bin/scan]: And here.

File: sonar-scanner/bin/scan (hunk 4 above)
Comment (underyx) [inline sonar-scanner/bin/scan]:
```suggestion
echo "======================"
```

File: sonar-scanner/bin/scan (hunk 5 above)
Comment (underyx) [inline sonar-scanner/bin/scan]:
```suggestion
echo "======================"
```

File: sonar-scanner/bin/scan [hunk 6]
```diff
@@ -21,10 +25,15 @@ sonar-scanner \
     2> >(tee -a /tmp/sonar-scanner-errors >&2)
 
 if grep -q 'A regular analysis is required before a branch analysis' /tmp/sonar-scanner-errors; then
+    echo
+    echo kiwicom/sonar-scanner
```
Comment (paveldedik) [inline sonar-scanner/bin/scan]: BTW how do you decide when to use quotes `"` and when not?
